lib/lhm/chunker.rb

```diff
@@ -25,7 +25,7 @@ def initialize(migration, connection = nil, options = {})
     def execute
       return unless @start && @limit
       @next_to_insert = @start
-      until @next_to_insert >= @limit
+      while @next_to_insert < @limit || (@next_to_insert == 1 && @start == 1)
```

Comment:
was the assumption here that the id of a table with a single record was always 1?